src/core/XMLprefs.cpp

```diff
@@ -361,6 +362,33 @@ int CXMLprefs::GetWithAttributes(const stringT &csBaseKeyName, const stringT &cs
   return iRevVal;
 }
 
+// Get a string value
+stringT CXMLprefs::GetWithAttributes(const stringT &csBaseKeyName, const stringT &csChildKeyName, 
```

Comment:
This is source code from one of the previous solutions and is no longer used. I've left the source code in case it might be useful in the future. I can also remove this if desired.